packages/components/src/suggestions/test/index.js

```diff
@@ -23,32 +27,38 @@ describe( '<Suggestions>', () => {
 	} );
 
 	test( 'render basic list of suggestions', () => {
-		const wrapper = shallow( <Suggestions { ...defaultProps } /> );
-		expect( wrapper.find( Item ) ).toHaveLength( 3 );
-		expect( wrapper ).toMatchSnapshot();
+		const { container } = render( <Suggestions { ...defaultProps } /> );
+
+		expect( screen.getAllByRole( 'button' ) ).toHaveLength( 3 );
+
+		expect( container.firstChild ).toMatchSnapshot();
```

Comment:
Is the snapshot test really providing value here?

Reply:
Yeah, checking for the values and the markup. Leaving it be, since it was there before.